tests/cluster.go

```diff
@@ -380,6 +380,11 @@ func (s *TestServer) GetTSOAllocatorManager() *tso.AllocatorManager {
 type TestCluster struct {
 	config  *clusterConfig
 	servers map[string]*TestServer
+	// tsPool is used to check the TSO uniqueness among the test cluster
+	tsPool struct {
+		sync.Mutex
+		pool map[uint64]struct{}
```

Comment:
Could we use `map[uint64]bool` instead since here we just need a `set`?

Reply:
Because the size of `struct{}` in Golang is zero, so it could be better than `bool` here technically?